scripts/buildWebsite.js

```diff
@@ -57,7 +58,8 @@ async function build() {
     resolutions: packageJSON.resolutions,
     browserslist: packageJSON.browserslist,
     scripts: {
-      build: "DOCS_ENV=production PARCEL_WORKER_BACKEND=process parcel build 'docs/*/*/docs/*.mdx' 'packages/dev/docs/pages/**/*.mdx'",
+      // Add a public url if provided via arg (for verdaccio prod doc website build since we want a commit hash)
+      build: `DOCS_ENV=production PARCEL_WORKER_BACKEND=process parcel build 'docs/*/*/docs/*.mdx' 'packages/dev/docs/pages/**/*.mdx' --no-scope-hoist ${publicUrlFlag}`,
```

Comment:
Why did we add `--no-scope-hoist` here?

Reply:
Ah, at the time `--no-scope-hoist` was already a part of this cmd line call from main. I see it isn't there anymore on main so I'll remove as well